### PR TITLE
8679 update expired email body

### PR DESCRIPTION
**Pull request by ozamani9**
*Issue #:* /bcgov/entity#8679

*Description of changes:*

Updated Email Body of Expired Email

### CODE REVIEW

**Comment by ozamani9** [comment]
The Expired Email Body was changed to contents from this spreadsheet.  

https://docs.google.com/spreadsheets/d/11wic6XEals7mLm7l64_Zl5K0YHuRAAfERpFPK--c4m0/edit#gid=0

**Review by cameron-freshworks — APPROVED** [review]
Code looks good!

**Review by vysakh-menon-aot — APPROVED** [review]

File: queue_services/entity-emailer/tests/unit/email_processors/test_nr_notification.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
         [{'name': 'TEST Company Name', 'state': 'NE'}, {'name': 'TEST2 Company Name', 'state': 'APPROVED'}]),
     ('before-expiry', 'NR 1234567', 'Expiring Soon', '2021-07-20T00:00:00+00:00', None, 'TEST3 Company Name',
         [{'name': 'TEST3 Company Name', 'state': 'CONDITION'}, {'name': 'TEST4 Company Name', 'state': 'NE'}]),
-    ('expired', 'NR 1234567', 'Expired', None, None, None,
+    ('expired', 'NR 1234567', 'Expired', None, None, 'TEST Expired Company Name',
```
Comment (argush3) [inline queue_services/entity-emailer/tests/unit/email_processors/test_nr_notification.py:36]: I think code needs to be added to the test to include a block where option is `expired` so you can actually check the expected legal name

Comment (ozamani9) [reply queue_services/entity-emailer/tests/unit/email_processors/test_nr_notification.py:36]: I had to create a new P.R. i saw this after the merge.

Please review new P.R.  1402

https://github.com/bcgov/lear/pull/1402

**Pull request merged**